iroha/src/account.rs

```diff
@@ -136,14 +136,25 @@ pub mod isi {
         RemoveSignatory(<Account as Identifiable>::Id, PublicKey),
     }
 
+    /// Enumeration of all legal Outputs for `AccountInstruction` execution.
```

Comment:
I think we should replace word `legal` with something different

Reply:
I suggest 'possible'